routes/feeds.js

```diff
@@ -46,6 +46,15 @@ router.get('/i_vote_comment', isAuth, feed.iVoteComment);
 router.get('/feed-chat/', isAuth, feed.getFeedChatService);
 router.get('/feed-chat/anonymous', isAuthAnonim, feed.getAnonymousFeedChatService);
 router.get('/feed-chat/:feedId', isAuth, feed.getOneFeedChatService);
+router.get('/feed-chat-v2/', isAuth, (req, res) =>
+  feed.getFeedChatService(req, res, {getNewMediaMessage: true})
+);
+router.get('/feed-chat-v2/anonymous', isAuthAnonim, (req, res) =>
+  feed.getAnonymousFeedChatService(req, res, {getNewMediaMessage: true})
+);
+router.get('/feed-chat-v2/:feedId', isAuth, (req, res) =>
+  feed.getOneFeedChatService(req, res, {getNewMediaMessage: true})
+);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new routes for version 2 of feed-chat have been added correctly. However, it's important to ensure that the `getNewMediaMessage` option is properly handled in the respective service functions (`getFeedChatService`, `getAnonymousFeedChatService`, and `getOneFeedChatService`). If not, this could lead to unexpected behavior or errors. 

Also, consider adding error handling middleware to these routes. This will help catch any errors that might occur during the execution of the request and send an appropriate response back to the client.

Here's a sample of how you can add error handling:

```diff
router.get('/feed-chat-v2/', isAuth, (req, res, next) =>
  feed.getFeedChatService(req, res, {getNewMediaMessage: true}).catch(next)
);
router.get('/feed-chat-v2/anonymous', isAuthAnonim, (req, res, next) =>
  feed.getAnonymousFeedChatService(req, res, {getNewMediaMessage: true}).catch(next)
);
router.get('/feed-chat-v2/:feedId', isAuth, (req, res, next) =>
  feed.getOneFeedChatService(req, res, {getNewMediaMessage: true}).catch(next)
);
```

This assumes that your Express app has an error handling middleware set up at the end of your middleware stack. If not, you should definitely consider adding one. It's a good practice to centralize error handling in your application.


<!-- This is an auto-generated comment by OSS CodeRabbit -->